lib/compat/wordpress-5.9/wp-enqueue-block-support.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
Given that the plan is to include this PR in a WordPress core minor release, we can not create a new file:
"A minor release is intended for bugfixes and enhancements that do not add new deployed files and are at the discretion of the release lead",

I guess we need to find an existing file where we can put this function.

Reply:
https://make.wordpress.org/core/handbook/about/release-cycle/releasing-minor-versions/

Reply:
Moved to `blocks.php`